peerpodconfig-ctrl/config/rbac/role.yaml

```diff
@@ -55,6 +55,27 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - apps
+  resources:
```

Comment:
Shall we use `resourceNames` to limit the resources instances here?

Reply:
@huoqifeng I updated to use the resourceName for the daemonset